app/models/court_application_defendant_summary.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+class CourtApplicationDefendantSummary
+  include ActiveModel::Model
+
+  attr_reader :defendant
+  attr_accessor :defendant_id
+
+  def initialize(attributes = {})
+    super
+    @defendant ||= Defendant.find(attributes[:defendant_id])
+  end
+
+  def to_builder
+    Jbuilder.new do |defendant_summary|
+      defendant_summary.defendantId defendant_id
+      if defendant.person?
+        defendant_summary.defendantNINO defendant.defendable.person.nationalInsuranceNumber
+        defendant_summary.defendantASN defendant.defendable.arrestSummonsNumber
+        defendant_summary.defendantFirstName defendant_first_name
+        defendant_summary.defendantMiddleName defendant_middle_name
+        defendant_summary.defendantLastName defendant_last_name
+        defendant_summary.defendantDOB defendant.defendable.person.dateOfBirth.to_date
+      else
+        defendant_summary.defendantName defendant_organisation_name
+      end
+      defendant_summary.dateOfNextHearing date_of_next_hearing
+      defendant_summary.proceedingsConcluded proceedings_concluded?
+      defendant_summary.masterDefendantId defendant.masterDefendantId
+      # Dummy subjectId until we build the link and unlik appeals links
+      defendant_summary.subjectId "4b463e6a-105b-433b-a88d-057d6e645bfb"
```

Comment:
When I pulled an example payload from the HMCTS dev environment, the `subjectId` was the same as the `masterDefendantId`. Normally `masterDefendantId` is the same as `defendantId`, so my guess is that `subjectId` is basically an alias for `defendantId`:

```json
  "subjectSummary": {
    "dateOfNextHearing": "2025-02-18T09:01:01.001Z",
    "defendantASN": "DTFL",
    "defendantDOB": "1982-06-30",
    "defendantFirstName": "John",
    "defendantLastName": "Smith",
    "masterDefendantId": "664ee07b-ac43-43ed-8abe-d4eb04459a6c",
    "offenceSummary": [],
    "subjectId": "664ee07b-ac43-43ed-8abe-d4eb04459a6c"
  }
  ```

Reply:
From my conversation from Ganga it is just a UUID for subjectSummary payload, we need to have more conversation in regards to this for clarity so for now I have just added a dummy uuid we dont actually use this currently in any of our flows so should be good.